scripts/extensions/sams/src/utils/ui.tsx

```diff
@@ -53,6 +53,18 @@ export function getIconTypeFromMimetype(mimetype: string) {
     }
 }
 
+export function getFileSizeFromHumanReadable(fileSize: number, dataUnit: string) {
+    if (dataUnit.startsWith('Bytes')) {
```

Comment:
It might be more precise if we used an enum for `dataUnit` rather then using strings, similar to how we use enums for `SET_STATE` | `ASSET_STATE` etc.

Then use these values as the `value` attribute to the options in the select field.